index.html

```diff
@@ -0,0 +1,1704 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous" />
+    <title>Newsweek</title>
+</head>
+
+<body>
+    <header class="fixed-top">
+        <div class="container-fluid red">
+            <div class="row align-items-center">
+                <div class="col-4 d-none d-md-none d-lg-block">
+                    <time class="text-white">Date</time>
+                </div>
+                <div class="col-4 col-sm-4 col-md-7  col-lg-4 d-sm-block d-md-block d-block ">
+                    <a href="#"><img class="h-75 w-100" src="images/Newsweek_Logo.svg.png" alt="logo"></a>
+                </div>
+                <div
+                    class="col-8 col-sm-8 col-md-5 col-lg-4 d-flex d-md-flex d-sm-flex flex-row-reverse align-items-center align-items-sm-center vertical-align-middle">
+                    <button class="navbar-toggler text-white d-lg-none  " type="button" data-toggle="collapse"
+                        data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon text-white">...</span>
+
+                    </button>
```

Comment:
##### Navigation bar
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/)). Your navbar is not properly sized and contains sub navbar on the medium screen.

**Yours**
<img width="854" alt="Screenshot 2020-01-03 at 17 12 15" src="https://user-images.githubusercontent.com/31304943/71734404-359b8c00-2e4c-11ea-80d8-f8f21275dcb1.png">


**Original**
<img width="981" alt="Screenshot 2020-01-03 at 17 11 27" src="https://user-images.githubusercontent.com/31304943/71734384-20266200-2e4c-11ea-9958-304fd9de994d.png">